examples/panda/panda_goal_reaching/controllers/robot_supervisor_manager/DDPG_runner.py

```diff
@@ -0,0 +1,85 @@
+from numpy import convolve, ones, mean, random
+
+from robot_supervisor_ddpg import PandaRobotSupervisor
+from agent.ddpg import DDPGAgent
+
+
+def run():
+    # Initialize supervisor object
+    env = PandaRobotSupervisor()
+
+    # The agent used here is trained with the DDPG algorithm (https://arxiv.org/abs/1509.02971).
+    # We pass (10, ) as numberOfInputs and (7, ) as numberOfOutputs, taken from the gym spaces
+    agent = DDPGAgent(alpha=0.000025, beta=0.00025, input_dims=[env.observation_space.shape[0]], tau=0.001, batch_size=64,  layer1_size=400, layer2_size=400, n_actions=env.action_space.shape[0]) 
+              
+    # agent.load_models() # Load the pretrained model
```

Comment:
Might be better if we have the option to pass a path at run function in order to load the pre-trained

Reply:
Ok, I will set an option to pass a path at run function.